datadog/threadstats/base.py

```diff
@@ -203,7 +203,7 @@ def histogram(self, metric_name, value, timestamp=None, tags=None, sample_rate=1
         """
         Sample a histogram value. Histograms will produce metrics that
         describe the distribution of the recorded values, namely the maximum,
-        median, average, count and the 95th percentile. Optionally, specify
+        average, count and the 95th percentile. Optionally, specify
```

Comment:
Thanks, looks like we support a couple that aren't here as well, should we call those out - https://github.com/DataDog/datadogpy/blob/master/datadog/threadstats/metrics.py#L95-L102 ?

Reply:
That sounds like a good idea. Thanks for checking!